services/notifier/notifier.go

```diff
@@ -441,7 +443,7 @@ func (n *Notifier) Subscribe(
 	n.background.group.Go(func() error {
 		defer close(jobsCh)
 
-		pollSleep := time.Duration(0)
+		pollSleep := atomic.NewDuration(time.Duration(0))
```

Comment:
I missing something here, why do we have to use atomic :atom: ?

I can see pollSleep being accessed by different go routines

Reply:
Thanks for pointing out. I didn't mean to push these changes. Reverting this.